ops.md

```diff
@@ -28,7 +28,27 @@ Consistently delivers systems and features that provide significant value to cus
 
 ### Domain Knowledge
 
-Mid-to-high level Linux system administration; proficient with UA operations tools/style and contributes to new or updated tooling solutions; can provide troubleshooting across most parts of the stack; able to solve majority operational issues with the stack without escalation; participate in an on-call rotation.
+Mid-level Linux system administration; proficient with UA operations tools/style; can provide troubleshooting across some parts of the stack; able to solve some operational issues without escalation; knows when to escalate to request assistance;  participates in in-hours on-call rotation.
```

Comment:
I wanted Ops Engs to be on daytime on-call because their triage skills are not across the whole stack.  I see it still as a learning position.